features/support/env.rb

```diff
@@ -34,14 +33,22 @@ def install_fixture_gems
   end
 end
 
-def current_ip
-  if OS.mac?
-    'host.docker.internal'
-  else
-    ip_addr = `ifconfig | grep -Eo 'inet (addr:)?([0-9]*\\\.){3}[0-9]*' | grep -v '127.0.0.1'`
-    ip_list = /((?:[0-9]*\.){3}[0-9]*)/.match(ip_addr)
-    ip_list.captures.first
+def remove_installed_gems
```

Comment:
Why is this done? Should we use gitignore instead?

Reply:
I didn't like the fact that multiple versions of the Gem could be installed under the `temp-bugsnag-lib` directory in each fixture.  I only noticed when running against 6.9.0 after running against 6.8.0, and had both `bugsnag-6.8.0` and `bugsnag-6.9.0` in the directory, and as there're many fixtures this is a bit of a clean up

Reply:
Can we add a comment with this clarification?

Reply:
Added